### PR TITLE
refactor(etls): improve euinvest - EUBFR-265

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

`third_parties` have been improved together with `project_locations`.

### CODE REVIEW

File: services/ingestion/etl/euinvest/csv/test/unit/lib/__snapshots__/transform.spec.js.snap [hunk 1]
```diff
@@ -178,3 +178,582 @@ The system will help resolve flooding problems and provide an outlet for unused
...
+  "project_id": "518",
+  "project_locations": Array [
+    Object {
+      "address": "",
+      "centroid": null,
+      "country_code": "DE",
+      "location": null,
+      "nuts": Array [],
+      "postal_code": "",
+      "region": "Emmendingen",
```
Comment (yhuard) [inline services/ingestion/etl/euinvest/csv/test/unit/lib/__snapshots__/transform.spec.js.snap]: `Emmendingen` is the town, `Baden-Württemberg` the region.

Same with `Albstadt-Tailfingen` (town).

I guess that if there are 3 parts in the location: part[0] = town, part[1] = region, part[2] = country code

Just an assumption from what I see

Comment (kalinchernev) [reply services/ingestion/etl/euinvest/csv/test/unit/lib/__snapshots__/transform.spec.js.snap]: Thanks for noticing! It turns out this part will need even more improvements.

Comment (kalinchernev) [reply services/ingestion/etl/euinvest/csv/test/unit/lib/__snapshots__/transform.spec.js.snap]: I moved existing logic to the lib module and corrected the known town/region cases.
I'll add a few more tests tomorrow in the lib to cover for a few more known edge cases.
